Initium-Odp/src/com/universeprojects/miniup/server/ODPDBAccess.java

```diff
@@ -4294,6 +4294,14 @@ public String doCharacterKilled(CachedEntity user, final CachedEntity characterT
 					attackingCharacter.setProperty("combatType", null);
 					attackingCharacter.setProperty("locationEntryDatetime", new Date());
 	
+					// Make the attacking character party no longer in combat mode
+					List<CachedEntity> party = getParty(ds, attackingCharacterFinal);
+					if (party != null) {
+						setPartiedField(party, attackingCharacter, "mode", ODPDBAccess.CHARACTER_MODE_NORMAL);
+						setPartiedField(party, attackingCharacter, "combatant", null);
+						setPartiedField(party, attackingCharacter, "combatType", null);
+						setPartiedField(party, attackingCharacter, "locationEntryDatetime", new Date());
```

Comment:
Check that setPartiedField() actually calls ds.put() on each of the party members too (my guess is it doesn't).  I think there is a method for putting all party members. Look for it where setPartiedField() was written, it would be near there.